opm/autodiff/SimulatorFullyImplicitBlackoilOutput.hpp

```diff
@@ -215,7 +216,7 @@ namespace Opm
                              const double* permeability );
 
         /** \copydoc Opm::OutputWriter::writeInit */
-        void writeInit();
+        void writeInit(const std::vector<data::CellData>& simProps = {} , const NNC& = NNC());
```

Comment:
This is the function declaration; so the arguments are *needed* - do you mean they should become normal not-defaulted arguments?